src/main/resources/data/retail-arrangements.json

```diff
@@ -0,0 +1,56 @@
+[
+  {
+    "BBAN": "123456",
+    "BIC": "bICOWgEl",
+    "accountHolderAddressLine1": "Pt0Ed",
```

Comment:
For being static data it seems very random. Could you give it more real life names?

Reply:
This is just a starting point, I am waiting on more data from product. Just making a PR to get ahead.